adobe_vipm/adobe/config.py

```diff
@@ -29,26 +33,50 @@ def __init__(self) -> None:
 
     @property
     def auth_endpoint_url(self) -> str:
+        """
+        Adobe auth endpoint URL.
+
+        Returns:
+            str: endpoint url
```

Comment:
Typing is not needed as we already have in the method. We can write just "The endpoint url"

```suggestion
        Returns:
            The endpoint url
```